CMakeLists.txt

```diff
@@ -21,7 +21,7 @@ download_project(PROJ                googletest
 # when building with Visual Studio
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 
-add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
+add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR} EXCLUDE_FROM_ALL)
```

Comment:
Interesting, how do you feel about checking for xcode and conditionally setting that flag? The question is whether it's worth it to add all this complexity for what is essentially a convenience feature; Since this is just an example project, which, above all, should be simple and easy to understand.